tests/functional/src/main/java/org/mule/functional/classloading/isolation/maven/RawDependencyGraphProvider.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.functional.classloading.isolation.maven;
+
+import java.util.List;
+
+/**
+ * Provides a dependency graph in .dot format.
```

Comment:
.dot -> DOT
Can you add a brief description of what is supposed to be on the DOT file?